registry/sql-registry/registry/models.py

```diff
@@ -674,16 +674,16 @@ def to_attr(self) -> ProjectAttributes:
 class SourceDef:
     def __init__(self,
                  name: str,
-                 path: str,
                  type: str,
                  qualified_name: str = "",
                  preprocessing: Optional[str] = None,
                  event_timestamp_column: Optional[str] = None,
                  timestamp_format: Optional[str] = None,
-                 tags: Dict = {}):
+                 tags: dict = {},
+                 **options: Dict[str, Any]):
         self.qualified_name = qualified_name
         self.name = name
-        self.path = path
```

Comment:
Why is `path` removed?

Reply:
It's now in options, and not limited to `path`.
The change is to support other source types, some of them don't have `path` attribute.